etc/config/kotlin.amazon.properties

```
@@ -39,3 +39,6 @@ compiler.kotlinc1500.java_home=/opt/compiler-explorer/jdk-16.0.1
 compiler.kotlinc1510.exe=/opt/compiler-explorer/kotlin-jvm-1.5.10/bin/kotlinc-jvm
 compiler.kotlinc1510.name=kotlinc 1.5.10
 compiler.kotlinc1510.java_home=/opt/compiler-explorer/jdk-16.0.1
+compiler.kotlinc1520.exe=/opt/compiler-explorer/kotlin-jvm-1.5.20/bin/kotlinc-jvm
+compiler.kotlinc1520.name=kotlinc 1.5.20
+compiler.kotlinc1520.java_home=/opt/compiler-explorer/jdk-16.0.1
```

We usually add a newline at EOL
```suggestion
compiler.kotlinc1520.java_home=/opt/compiler-explorer/jdk-16.0.1

```